internal/config/config.go

```diff
@@ -140,6 +140,22 @@ func (c *AuthConfig) Token(hostname string) (string, string) {
 	return token, source
 }
 
+// HasEnvOrConfigToken checks whether the current env or config contains a token
```

Comment:
```suggestion
// HasEnvOrConfigToken checks whether the current env or config contains a token.
```

Super super small nit, but I think Go doc format likes periods at the end of comments.